frontend/src/pages/ConnectedPages/components/axiosRequest.ts

```diff
@@ -0,0 +1,41 @@
+import axios, { AxiosRequestConfig } from "axios";
+import { getToken } from "../../../utils/auth";
+
+axios.defaults.baseURL = "http://localhost:5000";
+axios.defaults.headers.common["Authorization"] = `Bearer ${getToken()}`;
```

Comment:
We should probably in this file also check that the token is indeed valid, you can use the function `isTokenValid` from utils/auth for that